src/pages/openrefine_history.md

```diff
@@ -0,0 +1,60 @@
+---
+# OpenRefine History 
+
+Over the last ten years, OpenRefine has undergone significant transformations, including changes in its name and development model. It has transitioned from an enterprise-led model to a grant-based funding model.
+
+```mermaid
+%%{init: { 'theme': 'forest', 'timeline': {'disableMulticolor': true}}}%%
+timeline
+    title OpenRefine timeline
+    . : Project Name
+                        : Main Release
+                        : Development Model
+
+      2008 - 2010 : Freebase Gridworks
+                            : Version 1.0
+                            : Lead by Metaweb Technologies, Inc. with open source contributors
+
+    2010 - 2012 : Google Refine
+                    : Version 2.0 and 2.5
+                    : Lead by Google with open source contributors
+
+    2012 - 2018 : OpenRefine
+                    : Version 2.6
+                    : Solely with volunteer open source contributors
+
+      2018 - 2019 : OpenRefine
+                    : Version 3.0
+                    : First grants with Google News Initiatives and CZI with open source contributors
+
+      2020 - Today : OpenRefine
+                    : Version 4.x and 3.7.x
+                    : Grant based model with CS&S as fiscal sponsor with open source contributors
+```
+
+## 2008 - 2010: Freebase Gridworks
+<img src="/img/logo/freebase.png" alt="Freebase Gridworks" height="50"/>
+The history of OpenRefine dates back to May 2010, when Metaweb Technologies, Inc. initiated the development of an open-source data cleansing tool named Freebase Gridworks. Freebase Gridworks, conceived by David Huynh, simplified data cleaning and transformation data to import into Freebase, a large collaborative knowledge base built by Metaweb. Freebase Gridworks offered a user interface for editing and reconciling large datasets.
+
+## 2010 - 2012: Google Refine
+<img src="/img/logo/google-refine.png" alt="Google Refine" height="50"/>
+In July 2010, Google acquired Metaweb Technologies and with it, Freebase Gridworks. Following this acquisition, the tool was renamed "Google Refine". Under Google's stewardship, Google Refine 2.0 was released, featuring enhanced data cleaning capabilities. This version established the tool's reputation for handling various data-related tasks, such as cleaning messy data, transforming data formats, and extending data sets with web services.
+
+## 2012 - 2018: OpenRefine
+<img src="/img/logo/open-refine.png" alt="OpenRefine" height="50"/>
+In 2012, Google transferred leadership of the project to the open source community, who renamed it "OpenRefine" and moved it to GitHub. The following years saw OpenRefine continuing to gain a diverse user base, including scientists and researchers, journalists, semantic web practitioners, librarians, and cultural heritage specialists.
+
+In 2017, OpenRefine began integrating with Wikidata by creating a reconciliation endpoint and a process to publish data to the Wikimedia project.
```

Comment:
In a way, phrased like this it gives the impression that the Wikidata recon service is made by the OpenRefine project, and we've tried to keep the two things separate, so perhaps we could rather phrase it like this:
```suggestion
In 2017, OpenRefine began integrating with Wikidata thanks to the availability of a reconciliation endpoint and a process to publish data to the Wikimedia project.
```